v2/kafka-to-bigquery/src/main/java/com/google/cloud/teleport/v2/transforms/AvroTransform.java

```diff
@@ -79,6 +96,7 @@ public WriteResult expand(PCollection<KafkaRecord<byte[], byte[]>> kafkaRecords)
     Schema schema = SchemaUtils.getAvroSchema(avroSchema);
     WriteResult writeResult;
 
+    // TODO: Add support for error handler during BQ writes.
     Write<TableRow> writeToBQ =
```

Comment:
`writeToBQ` -> `writeToBq` or `writeToBigQuery`

Reply:
Done

Reply:
I changed this AvroDynamicTransform and forgot to change here. I changed it now.